docs/documentation/server_admin/topics/realms/email.adoc

```diff
@@ -113,11 +121,13 @@ Replace ApplicationId with the id of your application in Azure, usually a UUID.
 Auth Token ClientSecret::
 `<Secret configured>`
 
-== Configuration for Google Mail
+=== Configuration for Google Mail
 
-Not supported by Keycloak yet, because Google decided to not allow client-secrets for the Client Credentials Grant.
+Not supported by {project_name} yet, because Google decided to not allow client-secrets for the Client Credentials Grant.
```

Comment:
```suggestion
This feature is not  yet supported by {project_name}, because Google does not allow client-secrets for the Client Credentials Grant.
```